### ISSUE TITLE
Resource documentation is missing

### ISSUE THREAD

**Issue by smartinez290** [haffo/hit-iz-tool#71]
Add "Resource Documentation" to the IZtool "Documentation" section. Just like it was done in the MU-2 testing tool.

**Comment by smartinez290** [comment]
@robnist the tool does not have a Resource documentation tab -- the Installation Guide is included under the " User Documentation" and the "Tool Downloads" @haffo is going to include only the war files.  Are you ok with this approach?

**Comment by robnist** [comment]
Do exactly as we did for the 2014 Tools. 
A key design point of UIs is to keep the same workflow across tool iteration as much as possible. If there is no good reason to change then we don't. Here there is no good reason and the 2014 design worked very well. 

As a note, I have received numerous complaints about the profile view not being available as a main tab; even after I explained why we had to do this, users are still not happy.  People are sued to looking in a certain place for things, we want to keep it the same.

![image](https://cloud.githubusercontent.com/assets/14874904/11186461/d0251c08-8c4f-11e5-9727-ef2679ee08d4.png)

**Comment by haffo** [comment]
Rob,Do you want us to move the profile-viewer and value sets to upper level ?  Harold

**Comment by robnist** [comment]
No. We did that at first and I agreed to the change because I think overall it makes the most sense given the new workflow (with more functionality). So there was a good enough reason.

I’m just stating that some user are not happy and I’m trying to convince why we had to. This is just a point I’m making that our documentation page needs to stay the same, users know where to look for things. We want to be consistent across tools as much as possible.
